Gemfile

```diff
@@ -31,3 +31,8 @@ group :development do
   gem 'spring',                                 '~> 2.0'
   gem 'spring-watcher-listen',                  '~> 2.0'
 end
+
+group :development, :test do
+  gem 'factory_bot_rails'
+  gem 'rspec-rails', '~> 4.0.1'
+end
```

Comment:
Please add similar version constraints as the other gems have, and indent the constraint properly.